src/transformers/trainer.py

```diff
@@ -441,9 +441,13 @@ def __init__(
                 self.backward_prefetch = BackwardPrefetch.BACKWARD_POST
 
             self.forword_prefetch = False
-            if "forword_prefetch" in self.args.fsdp_config and self.backward_prefetch:
+            if self.args.fsdp_config.get("forword_prefect", False):
```

Comment:
Just checking here with @pacman100 to make user using `backward_prefetch` was a mistake

Reply:
yes, this fix is correct